Ginger/GingerCore/SourceControl/SVNSourceControl.cs

```diff
@@ -258,10 +259,14 @@ public override bool GetLatest(string path, ref string error, ref List<string> c
         {
             if (client == null) Init();
             SvnUpdateResult result;
+
+            RepositoryFolderBase repositoryFolderBase = WorkSpace.Instance.SolutionRepository.GetRepositoryFolderByPath(Path.GetDirectoryName(path));
             try
             {
                 mConflictsPaths.Clear();
```

Comment:
move this code inside a function and reuse newly added function wherever you need to call update file